dbmigrate-lib/src/drivers/sqlcipher.rs

```diff
@@ -0,0 +1,124 @@
+use url::Url;
+
+use sqlcipher_client::Connection;
+
+use super::Driver;
+use errors::{Result, ResultExt};
+
+const SQLCIPHER_DEFAULT_CIPHER: &'static str = "AES-256-CBC";
+
+#[derive(Debug)]
+pub struct Sqlcipher {
+    conn: Connection,
+}
+
+impl Sqlcipher {
+    pub fn new(url: &str) -> Result<Sqlcipher> {
+        // Expect a URL in the form:
+        // sqlcipher://user:password@host/path/to/file.db
+        //
+        // The user and host components are not used.
+        // The password is used with the PRAGMA KEY statement.
+        let parsedurl = Url::parse(url)
```

Comment:
can you rename `parsedurl` to `parsed_url`?

Reply:
No problem.  Does Rust use the same words_with_underscores naming convention for both local variables and methods?

Reply:
Rust uses snake_case pretty much everywhere except struct/trait name